components/at_mention/at_mention.jsx

```diff
@@ -103,7 +103,7 @@ export default class AtMention extends React.PureComponent {
                 >
                     <Pluggable>
                         <ProfilePopover
-                            user={user}
+                            userId={user.id}
```

Comment:
`Pluggable` component is a bit confusing to me but if I understand it correctly, it may render the registered plugin component iff `pluggableName` props is passed into it.  If not, then it will render its children.
So in this case, I presume that it will render the original (`ProfilePopover`).

I can't exactly confirm but I think this change won't break the plugin but in case I'm mistaken and there's a need to pass a `user` object, it should be passed into the `Pluggable` component itself and not rely props from its children like `ProfilePopover` in this case.

Reply:
I'm pretty sure these props are passed into the ProfilePopover or whatever custom component replaces it, so I don't think we can change this easily.